glide.lock

```diff
@@ -667,4 +728,61 @@ imports:
   - tools/clientcmd/api
   - tools/metrics
   - transport
-testImports: []
+testImports:
```

Comment:
@jangie can you make a commit on top of this one that removes `integration/glide.{yaml,lock}` and `integration/vendor` to see if it's still green ? 👼